docs/ibm-cloud-rules.md

```diff
@@ -360,13 +360,13 @@ or <code>application/merge-patch+json</code>.</td>
 <td>oas3</td>
 </tr>
 <tr>
-<td><a href="#ibm-path-param-not-crn">ibm-path-param-not-crn</a></td>
+<td><a href="#ibm-path-parameter-crn">ibm-path-parameter-crn</a></td>
```

Comment:
I think this is a case where we'd want to leave "not" in the name or name it something like `ibm-no-crn-path-parameters` since a CRN path param would be the problem but a path param not being a CRN is the rule

Reply:
Yeah, I agree... I think I might have taken the new rule naming conventions too far or maybe misinterpreted them :)

I'll change it to `ibm-path-parameter-not-crn` (i.e. net change is just `param` -> `parameter`) if that sounds right.

Reply:
Yep, that sounds good to me!

Reply:
pushed an amended commit that renames the rule to be `ibm-path-parameter-not-crn`